irohad/ametsuchi/impl/flat_file_block_storage.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_FLAT_FILE_BLOCK_STORAGE_HPP
+#define IROHA_FLAT_FILE_BLOCK_STORAGE_HPP
+
+#include "ametsuchi/block_storage.hpp"
+#include "ametsuchi/impl/flat_file/flat_file.hpp"
```

Comment:
```C++

#include "ametsuchi/block_storage.hpp"

#include "ametsuchi/impl/flat_file/flat_file.hpp"
#include "interfaces/iroha_internal/block_json_converter.hpp"
#include "logger/logger_fwd.hpp"
```
Because the class implements `BlockStorage`.